corehq/apps/reports/extension_points.py

```diff
@@ -0,0 +1,25 @@
+from corehq.extensions import extension_point, ResultFormat
+
+
+def customize_user_query(user, domain, user_query):
+    for mutator in user_query_mutators(domain):
+        user_query = mutator(user, domain, user_query)
+    return user_query
+
+
+@extension_point(result_format=ResultFormat.FLATTEN)
+def user_query_mutators(domain):
```

Comment:
Hey @snopoke
Can you please guide me on why we have passed `domain` here?

Reply:
So that we can configure them per domain?
Currently we just need this for "icds-cas" so we should update that in the icds PR.

Reply:
https://github.com/dimagi/commcare-icds/pull/65/files#r479105471